### ISSUE TITLE
A primary user can be created without an org

### ISSUE THREAD

**Issue by JMurphyWeb** [emfoundation/ce100-app#577]
An admin can create a primary user without an organisation. 

When this user activates their account, the app will break if they click on their profile in the navbar and if they try to create a challenge. Should there be some validation on the 'Create User' form that says if it is a primary user being created, then they have to be attached to an organisation?

Question for @asimonw.

**Comment by JMurphyWeb** [comment]
Linked to the org dropdown when creating a user issue: https://github.com/emfoundation/ce100-app/issues/319

**Comment by asimonw** [comment]
@JMurphyWeb Yes, ideally there would be some validation. Before saving a primary or secondary user, check that the org field is populated, thanks!

**Comment by asimonw** [comment]
@JMurphyWeb Thanks for putting the validation in place. The error message is a bit cryptic though: "Org id must be larger than or equal to 1." Could we change that to "Please select an organisation"?
